.github/workflows/unit_tests.yml

```diff
@@ -75,6 +75,10 @@ jobs:
         mne sys_info
     - name: Install MNE-BIDS
       run: pip install --no-deps .
+    - name: Install pandas
+      if: "matrix.os == 'ubuntu-latest'"
+      run: |
+        pip install -U pandas
```

Comment:
I would also list `pandas` in the installation page as an optional dependency

Reply:
done